parachain/pallets/inbound-queue/src/test.rs

```diff
@@ -39,6 +46,13 @@ fn test_submit_happy_path() {
 			],
 		}
 		.into()]);
+
+		let reward = Parameters::get().rewards.local;
+		assert!(Balances::balance(&relayer) >= reward, "relayer was rewarded");
+		assert!(
+			Balances::balance(&channel_sovereign) <= initial_fund - reward,
+			"sovereign account paid reward"
+		);
```

Comment:
```suggestion
		let delivery_cost = InboundQueue::calculate_delivery_cost(message.encode().len() as u32);
		assert!(Parameters::get().rewards.local < delivery_cost, "delivery cost exceeds pure reward");
		assert_eq!(Balances::balance(&relayer), delivery_cost, "relayer was reimbursed and rewarded");
		assert!(
			Balances::balance(&channel_sovereign) <= initial_fund - delivery_cost,
			"sovereign account paid reward and delivery fee"
		);
```

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/1086